_includes/operator/examples.md

```diff
@@ -0,0 +1,147 @@
+# Cluster Operator examples
+
+Before deploying a StorageOS cluster, create a Secret to define the StorageOS
+API Username and Password in base64 encoding.
+
+```bash
+kubectl create -f - <<END
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "storageos-api"
+  namespace: "default"
+  labels:
+    app: "storageos"
+type: "kubernetes.io/storageos"
+data:
+  # echo -n '<secret>' | base64
+  apiUsername: c3RvcmFnZW9z
+  apiPassword: c3RvcmFnZW9z
+END
+```
+
+
+Create a `cluster-config.yaml` according to your needs from the examples below.
+
+```bash
+kubectl create -f cluster-config.yaml
+```
+
+Note that StorageOS will be deployed in the storageos NameSpace, irrespective
+of any NameSpace defined within the CR yaml.
```

Comment:
StorageOS namespace isn't fixed. The cluster-operator can install storageos in any namespace. This can be configured by setting `spec.namespace`.